dotcom-rendering/src/components/Discussion.tsx

```diff
@@ -66,7 +66,7 @@ export const Discussion = ({
 	user,
 	idApiUrl,
 }: Props) => {
-	const [commentPage, setCommentPage] = useState<number>();
+	const [page, setPage] = useState<number>(1);
```

Comment:
Why the rename?

We may be on the “third” page of a liveblog, but on the “first” comment page, for example

Reply:
Yep I did think this. I went for page as that's how its referred to the rest of the way through the tree so I thought it made it easier to follow. Happy to use commentPage though if thats more explicit